frontier/.htaccess

```diff
@@ -8,57 +8,67 @@ AddType application/rdf+xml .owl
 AddType text/turtle .ttl
 AddType application/n-triples .n3
 AddType application/ld+json .json
-# Rewrite engine setup
-RewriteEngine On
 
-# Rewrite rule to serve HTML content from the vocabulary URI if requested
-RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
-RewriteCond %{HTTP_ACCEPT} text/html [OR]
-RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
-RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^def$ doc/index-en.html [R=303,L]
+RewriteEngine on
+
+# Rewrite rule for latest version.
 RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
 RewriteCond %{HTTP_ACCEPT} text/html [OR]
 RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
 RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
-RewriteRule ^/(.+) https://physta.github.io/frontier/OnToology/Frontier.ttl/documentation/doc/index-en.html#$1 [R=303,NE,L]
+RewriteRule ^$ https://physta.github.io/frontier/OnToology/Frontier.ttl/documentation/doc/index-en.html [R=303,L]
```

Comment:
This one is redundant with the one in line 46. You can group them by doing:
RewriteRule ^(.*)$ https://physta.github.io/frontier/OnToology/Frontier.ttl/documentation/doc/index-en.html [R=303,L]